unit_tests/test_tlslite_extensions.py

```diff
@@ -22,7 +22,7 @@
         RenegotiationInfoExtension, ALPNExtension, StatusRequestExtension, \
         SupportedVersionsExtension, VarSeqListExtension, ListExtension, \
         ClientKeyShareExtension, KeyShareEntry, ServerKeyShareExtension, \
-        CertificateStatusExtension
+        CertificateStatusExtension, HRRKeyShareExtension
```

Comment:
The HRR is very cryptic (even the TLS document doesn't expand this acronym). Maybe having a comment somewhere expanding this HRR would be useful for a future reader of the code.